modules/incheal.lua

```diff
@@ -126,7 +126,9 @@ function IncHeal:PositionBar(frame, incAmount)
 end
 
 function IncHeal:Update(frame)
-	local amount = (HealComm:GetHealAmount(frame.unitGUID, HealComm.ALL_HEALS) or 0) * (HealComm:GetHealModifier(frame.unitGUID) or 1)
+	local castedHeals = (HealComm:GetHealAmount(frame.unitGUID, HealComm.CASTED_HEALS) or 0)
+	local nextHotTick = (HealComm:GetHealAmount(frame.unitGUID, HealComm.HOT_HEALS, GetTime() + 3) or 0)
```

Comment:
```suggestion
	local nextHotTick = (HealComm:GetHealAmount(frame.unitGUID, HealComm.OVERTIME_HEALS, GetTime() + 3) or 0)
```
Unfortunately not all overtime heals tick every 3 seconds, so this won't be completely accurate

Reply:
```
HealComm:GetHealAmount(guid, healFlags[, timeFrame[, casterGUID]])
Returns the healing amount using the passed filters.
For hots with a time frame, it will return the ticks. For example Rejuvenation ticks for 3,000 at 3s, 6s, 9s, 12s and you retrieve only the last 6 seconds then you will get 6,000.

Parameters
guid
GUID of the player to retrieve healing incoming on
healFlags
Bit flag for the heal type, lets you retrieve a specific type of heals, see Bit Fields for more information.
timeFrame
Retrieves all heals that are going to end or tick before the time period given, when nil is passed it gets all pending heals.
casterGUID
Retrieves heals only from the passed GUID, when nil is passed then it gets pending heals from all casters.
Return value
Returns the total amount of healing that is going to be done using the provided filters, if no heaing is incoming using the given filters then nil is returned.

Examples
Retrieve all heals that will be cast on the player within the next 5 seconds.

   local HealComm = LibStub("LibHealComm-4.0")
   local heal = HealComm:GetHealAmount(UnitGUID("player"), HealComm.ALL_HEALS, GetTime() + 5) or 0
   print(string.format("You have at least %d healing incoming in the next 5 seconds.", heal))
Retrieve only direct and channeled heals in the next 2 seconds, ignores hots.

   local HealComm = LibStub("LibHealComm-4.0")
   local heal = HealComm:GetHealAmount(UnitGUID("player"), HealComm.CASTED_HEALS, GetTime() + 2) or 0
   print(string.format("You have at least %d healing incoming through casted and channeled heals in the next 2 seconds.", heal))```

Unless I'm misunderstanding the documentation. Given that it's the newer version that may be the case, but in testing it seems to function that way

Reply:
I basically followed their example `For hots with a time frame, it will return the ticks. For example Rejuvenation ticks for 3,000 at 3s, 6s, 9s, 12s and you retrieve only the last 6 seconds then you will get 6,000.`

Reply:
Sorry if my comment was a bit confusing - since you are checking hot heals and not overtime heals you will be missing out on channeled heals like Tranquility. Not only this, but Tranquility ticks every 2 seconds and not every 3, which I believe could result in you displaying 2 ticks as incoming at times.

So my suggested change here is to use OVERTIME_HEALS over HOT_HEALS. For the second part I'm not sure how big of a deal it is.

Reply:
Hrm I'll look more into it, but I think as far as channeled heals they're captured in the first value with `CASTED_HEALS`. So in my fix it would still show the total channeled heal. So maybe I need to split into direct, hots, and channeled ... Happy to tweak though